build/debug_keybase/main.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"crypto/md5" // nolint:gosec // Weak hashing function only used to check if the file has been changed
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/korovkin/limiter"
+	"gopkg.in/yaml.v2"
+
+	"github.com/pokt-network/pocket/app/client/keybase"
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/utils"
+)
+
+const (
+	// NOTE: This is the number of validators in the private-keys.yaml manifest file
+	numValidators = 999
+
+	// Increasing this number is linearly proportional to the amount of RAM required for the debug client to start and import
+	// pre-generated keys into the keybase. Beware that might cause OOM and process can exit with 137 status code.
+	// 4 threads takes 350-400MiB from a few tests which sounds acceptable.
+	debugKeybaseImportConcurrencyLimit = 4
+)
+
+func main() {
+	if len(os.Args) < 3 {
+		fmt.Println("Usage: go run main.go <source_yaml> <target_folder>")
```

Comment:
`global.logger.Fatal()` ? 

Ditto elsewhere

Reply:
I disagree. This is something that's either ran by a human or by a pipeline, stdout and stderr are the way IMHO.